submission/submission-webapp/src/test/kotlin/ac/uk/ebi/biostd/files/web/UserFilesResourceTest.kt

```diff
@@ -1,59 +0,0 @@
-package ac.uk.ebi.biostd.files.web
-
-import ac.uk.ebi.biostd.factory.TestSuperUser
-import ac.uk.ebi.biostd.files.service.UserFilesService
-import ac.uk.ebi.biostd.files.web.common.FilesMapper
-import ac.uk.ebi.biostd.files.web.resources.UserFilesResource
-import ac.uk.ebi.biostd.resolvers.TEST_USER_PATH
-import ac.uk.ebi.biostd.resolvers.TestBioUserResolver
-import ac.uk.ebi.biostd.resolvers.TestUserPathResolver
-import ebi.ac.uk.security.integration.model.api.SecurityUser
-import ebi.ac.uk.test.createFile
-import io.github.glytching.junit.extension.folder.TemporaryFolder
-import io.github.glytching.junit.extension.folder.TemporaryFolderExtension
-import io.mockk.clearAllMocks
-import io.mockk.every
-import io.mockk.impl.annotations.MockK
-import io.mockk.junit5.MockKExtension
-import io.mockk.slot
-import org.junit.jupiter.api.AfterEach
-import org.junit.jupiter.api.Test
-import org.junit.jupiter.api.extension.ExtendWith
-import org.springframework.http.MediaType
-import org.springframework.test.web.servlet.get
-import org.springframework.test.web.servlet.setup.MockMvcBuilders
-
-@ExtendWith(MockKExtension::class, TemporaryFolderExtension::class)
```

Comment:
Now that both classes are merged, shouldn't these test cases go to a `PathFilesServiceTest` class?